src/components/common/Dialogs/RemoveSafeDialog/SafeListItem.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { SAFE_NETWORKS } from '~constants';
+import Avatar from '~shared/Avatar';
+import { HookFormCheckbox as Checkbox } from '~shared/Fields';
+import MaskedAddress from '~shared/MaskedAddress';
+import InvisibleCopyableAddress from '~shared/InvisibleCopyableAddress';
+
+import styles from './SafeListItem.css';
+
+const displayName = 'common.RemoveSafeDialog.SafeListItem';
+
+const MSG = defineMessages({
+  copyMessage: {
+    id: `${displayName}.copyMessage`,
+    defaultMessage: 'Click to copy Safe address',
+  },
+  safeNamePlaceholder: {
+    id: `${displayName}.safeNamePlaceholder`,
+    defaultMessage: 'Unknown',
+  },
+});
+
+interface Props {
+  safe: any;
+  isChecked: boolean;
+}
+
+const SafeListItem = ({ safe, isChecked }: Props) => {
+  const { formatMessage } = useIntl();
+  const safeNetwork = SAFE_NETWORKS.find(
+    (network) => network.chainId === Number(safe.chainId),
+  );
+  return (
+    <div className={`${styles.main} ${isChecked && styles.checked}`}>
```

Comment:
We could use the `classnames` library here and write this in a more readable way:
```
classnames(styles.main, { [styles.checked]: isChecked })
```